### PR TITLE
Run RPMInspect in LDAP JDK CI

### PR DESCRIPTION
**Pull request by ckelleyRH**

### CODE REVIEW

**Review by edewata — APPROVED** [review]
Same suggestion as in the tomcatjss PR. Feel free to update/merge.

File: tests/bin/rpminspect.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+# Don't run metadata check as we can't know the build host subdomain
+# of CI runners in advance to add to an allow list
+
+echo "Running RPMInspect on SRPM"
+rpminspect-fedora -E metadata /github/home/build/ldapjdk/SRPMS/*.rpm
```
Comment (fmarco76) [inline tests/bin/rpminspect.sh]: This (and line 10) could be relative path.

**Review by fmarco76 — APPROVED** [review]
LGTM. Only the small change for the path as for the other PRs.

**Comment by ckelleyRH** [comment]
Updated, thanks @edewata @fmarco76 !

**Pull request merged**